tests/pipert/core/utils/test_logging_modifiers.py

```diff
@@ -0,0 +1,30 @@
+import sys
+import pytest
+import logging
+from src.pipert2.utils.logging_module_modifiers import add_pipe_log_level, \
+    PIPE_INFRASTRUCTURE_LOG_LEVEL_NAME, PIPE_INFRASTRUCTURE_LOG_LEVEL
+
+
+@pytest.fixture()
+def pipe_logger():
+    add_pipe_log_level()
+    logger = logging.getLogger()
+    logger.addHandler(logging.StreamHandler(sys.stdout))
+    logger.setLevel(logging.WARNING)
+    return logger
+
+
+def test_plog_works_when_using_his_level(caplog, pipe_logger):
```

Comment:
San proposed a new format for test naming -
{action}_{test case}_{expected result}